apps/scan/frontend/src/screens/setup_scanner_screen.tsx

```diff
@@ -1,25 +0,0 @@
-import { CenteredLargeProse, H1, P, appStrings } from '@votingworks/ui';
-import { ScreenMainCenterChild } from '../components/layout';
-
-interface Props {
-  scannedBallotCount?: number;
-}
-
-export function SetupScannerScreen({ scannedBallotCount }: Props): JSX.Element {
```

Comment:
Is this related to the ballot bag? Looks like it's just an old file that was replaced by `internal_connection_problem_screen`?

Reply:
No, it's not related. I just found that it was unused when I was using knip to make sure I had removed everything related to the ballot bag feature.